### ISSUE TITLE
Investigate usage of ME0 related code in CMSSW, and possible cleanup of obsolete files

### ISSUE THREAD

**Issue by perrotta** [cms-sw/cmssw#31424]
There could be some ME0 labelled code in CMSSW which is not in usage any more, having possibly been incorporated in the "standard" GEM code.

Muon POG (@ArnabPurohit @trocino) and GEM ( @jshlee @watson-ij) reco contacts are [investigating](https://github.com/cms-sw/cmssw/pull/31386#issuecomment-690110039) about it

This issue is just to take note of it, and keep track of the advancements

**Comment by perrotta** [comment]
assign reconstruction

**Comment by perrotta** [comment]
> @silviodonato While not reconstruction, the ME0 pseudo digis and pseudo segments for the L1-trigger (https://github.com/cms-sw/cmssw/blob/master/L1Trigger/L1TGEM/python/me0TriggerDigis_cff.py#L8) will be phased out when we consolidate GEM and ME0.

Thank you @dildick . Indeed this will quite likely span several areas, I guess.
I add assignment to L1, simulation as well

**Comment by perrotta** [comment]
assign l1, simulation

**Comment by dildick** [comment]
Yes, many more areas will be affected, including Digitization, Trigger, Reconstruction, Validation and Visualization. Within the GEM group we have a good idea what needs to be done.

**Comment by jpata** [comment]
type muon

**Comment by jpata** [comment]
@cms-sw/muon-pog-l2 @CeliaFernandez @trocino
@cms-sw/gem-dpg-l2 @jshlee @watson-ij

is there any update on this?

**Comment by cecilecaillol** [comment]
@giovanni-mocellin Can you please followup on that?
